docs/sagemaker/train.md

```diff
@@ -99,6 +99,12 @@ _Note that SageMaker doesn’t support argparse actions. For example, if you wan
 
 Look [here](https://github.com/huggingface/notebooks/blob/master/sagemaker/01_getting_started_pytorch/scripts/train.py) for a complete example of a 🤗 Transformers training script.
 
+## Training Output Management
+
+`output_dir` defaults to '/opt/ml/model' which saves both checkpoints and models to '/opt/ml/model', and results in a large output `tar.gz` containing all checkpoint files and model. A large artifacts most of them possibly useless for deployment.
+
```

Comment:
```suggestion
You can control how checkpoints, logs, and artifacts are saved by customization the [TrainingArguments](https://huggingface.co/docs/transformers/master/en/main_classes/trainer#transformers.TrainingArguments). For example by providing `save_total_limit` as `TrainingArgument` you can control the limit of the total amount of checkpoints. Deletes the older checkpoints in `output_dir` if new ones are saved and the maximum limit is reached.


```

Reply:
why did do resolve this comment?

Reply:
Sorry, I have overlooked it and clicked by mistake.